### DISCUSSION TITLE
Inquiry about Auto-Type Feature in KeepassXC

### DISCUSSION THREAD

**Discussion by comiluv** [keepassxreboot/keepassxc#9625] [Q&A] [answered]
I am writing to inquire about the availability of an auto-type feature in KeepassXC. 
 In the past, I have experienced some embarrassing moments with other password managers, where the auto-type feature would mistakenly input my password in search bars, navigation forms, and other unintended fields.
 To address this issue, I started using the CheckPasswordBox plugin for Keepass2, which has provided me with a high level of satisfaction.
 However, I am curious to know if KeepassXC offers a similar feature. I have thoroughly reviewed the documentation but was unable to locate the placeholder for detecting password boxes during auto-typing. 
 If KeepassXC does indeed have this feature, I kindly request your guidance on how to utilize it effectively. Any information or instructions you can provide would be greatly appreciated.
 Thank you for your time and assistance. I look forward to your response.

**Comment by droidmonkey** [comment, marked as answer]
We do not have the "check password box" feature. We do recommend you use our browser extension for website filling.
